workbooks/migrations/0001_initial.py

```diff
@@ -147,9 +147,9 @@ class Migration(migrations.Migration):
             preserve_default=True,
         ),
         migrations.AddField(
-            model_name='worksheet',
-            name='plot',
-            field=models.ForeignKey(blank=True, to='workbooks.Worksheet_plot', null=True),
+            model_name='worksheet_plot',
+            name='worksheet',
+            field=models.ForeignKey(blank=True, to='workbooks.Worksheet', null=True),
```

Comment:
These changes need to be a different migration. Migration files should never really be changed. I'll fix it before the merge